src/rzslider.js

```diff
@@ -450,7 +450,9 @@
           var index = 0
           for (var i = 0; i < this.options.stepsArray.length; i++) {
             var step = this.options.stepsArray[i]
-            if (step === modelValue) {
+            var lastStep = i > 0 ? this.options.stepsArray[i - 1] : 0;
+
+            if (step === modelValue || (modelValue > lastStep && modelValue < step)) {
```

Comment:
some context require here, will you add comments here

Reply:
If a stepsArray is given in the options and the value is set from the outside to a value that is not part of the stepsArray, currently the slider will jump to the first element in the stepsArray. I changed it so that if the value is between two values the slider jumps to the higher value.

Reply:
I like this feature. However, could you add a test case for it?